### ISSUE TITLE
Unable to resolve exports path in package.json

### ISSUE THREAD

**Issue by MuTsunTsai** [glromeo/esbuild-sass-plugin#61]
Not sure if this is an issue with esbuild in general, or if there's anything that can be done with the plugins.

Take the package `swiper` as an example. In its package.json it lists all its exports such as 
```json
{
    "exports": {
        ...
        "./scss": "./swiper.scss",
        ...
    }
}
````
So if I write `import "swiper/scss";` it will resolve to "swiper/swiper.scss". And also I couldn't use the latter instead, or it will give me an error saying that this path is not exported by the package.
Now the problem is, since "swiper/scss" is not a path ending in ".scss", this plugin fails to intercept such importing. I could, of course, manually add it to the filter to make it work, but that wouldn't be practical if there're many such imports in my project. The plugin should be able to intercept, besides explicitly written paths, resolved path as well.

**Comment by glromeo** [comment]
@MuTsunTsai thank you for the best issue ever!
It helped me understand that I am still intercepting the onResolve which was a leftover of the v1.x design
and that is no longer necessary and actually doesn't work well given `esbuild` package exports resolution.
Getting rid of onResolve will help me simplify the plugin and get rid of the "namespaces".
Unfortunately the changes require some time to test the effects thoroughly but I hope to have v2.1.0 soon

**Comment by MuTsunTsai** [comment]
Oh, by the way, I'm not sure if this is related or should be post as a seperate issue, but I believe that this plugin also went wrong if the imported .scss file makes another import, and if the full path of the latter file contains non-ascii characters.

**Comment by MuTsunTsai** [comment]
Confirmed that this is fixed in v2.1.0.
